test/setup/TEST_JSON_COLUMN_FUNCTION.sql

```diff
@@ -0,0 +1,58 @@
+-- TEST_DELETE_LOGS.sql
```

Comment:
```suggestion
-- TEST_COLUMN_ARRAY_TO_COLUMN_LIST.sql
```
lets name it after the function we're testing: TEST_COLUMN_ARRAY_TO_COLUMN_LIST.sql